manim/utils/tex_file_writing.py

```diff
@@ -42,67 +45,88 @@ def generate_tex_file(expression, tex_template, source_type):
     return result
 
 
-def tex_to_dvi(tex_file, use_ctex=False):
-    result = tex_file.replace(".tex", ".dvi" if not use_ctex else ".xdv")
+def tex_compilation_command(compiler, tex_file, tex_dir):
+    if compiler["command"] in {"latex", "pdflatex", "luatex", "lualatex"}:
+        commands = [
+            compiler["command"],
+            "-interaction=batchmode",
+            '-output-format="' + compiler["output_format"][1:] + '"',
+            "-halt-on-error",
+            '-output-directory="{}"'.format(tex_dir),
+            '"{}"'.format(tex_file),
```

Comment:
Here also using fstrings instead of format would be good.

Reply:
I just took what was already there. This is not my code.
https://github.com/ManimCommunity/manim/blob/master/manim/utils/tex_file_writing.py#L56

Reply:
Changing it doesn't hurt.

Reply:
but if you suggest a reformatting, I'll of course gladly click on "accept"

Reply:
Sure here you go.
```suggestion
            f'-output-directory="{tex_dir}"',
            f'"{tex_file}"',
```